app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -56,17 +84,13 @@ def get_all_records():
     url = AIRTABLE_REQUEST_URL.format(AIRTABLE_BASE_ID)
     url = _add_fields_to_url(url)
     url += '&filterByFormula={ETL Included}=1'
-    headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
-
-    # Make request and retrieve records in json format
-    r = requests.get(url, headers=headers)
-    data = r.json()
+    data = _airtable_get_request(url)
 
     # Try to get records from data if the request was successful
     try:
         # If offset was included in data, retrieve additional paginated records
         if 'offset' in list(data.keys()):
-            request_info = [url, headers]
+            request_info = [url, {'Authorization': 'Bearer {}'.format()}]
```

Comment:
why did you remove headers? ```'Authorization': 'Bearer {}'.format()``` is not actually doing a replace because you didn't specify anything the format function

Reply:
ah oops, forgot to add the `AIRTABLE_API_KEY` for that one, just fixed it